code/modules/client/loadout/loadout_accessories.dm

```diff
@@ -197,7 +197,12 @@
 
 /datum/gear/accessory/mask
 	subtype_path = /datum/gear/accessory/mask
-	cost = 1500
```

Comment:
why raise the price of all masks instead of just the gondola mask

Reply:
wait shit I did that on the wrong one